patches/api/0417-Add-PlayerPreRespawnLocationEvent.patch

```diff
@@ -0,0 +1,69 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cryptite <cryptite@gmail.com>
+Date: Mon, 1 May 2023 15:23:34 -0500
+Subject: [PATCH] Add PlayerPreRespawnLocationEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerPreRespawnLocationEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerPreRespawnLocationEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..b321cc19d6ca57050233339ced6c84d9f793f1b6
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerPreRespawnLocationEvent.java
+@@ -0,0 +1,57 @@
++package com.destroystokyo.paper.event.player;
```

Comment:
This is not the right package for the new events, should be io.papermc.paper.event